doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -72,4 +72,11 @@ Access GeoServer front page
 ---------------------------
 
 * After a few seconds, GeoServer should be accessible at: `<http://localhost:8080/geoserver>`_
-* The default ``admin`` password is ``geoserver``.
\ No newline at end of file
+* The default ``admin`` password is ``geoserver``.
+
+Run GeoServer from Intellij on Windows
+---------------------------
+
+#. Add bash to your windows environment path and restart intellij.  
+#. Rebuild the security-tests project in the security module.  Right click on the security-tests project and click Rebuild.  Otherwise,  there will be symbol errors such as "cannot find symbol AbstractUserGroupServiceTest".
+#. Using wcs1_1 as the working directory, run a mvn clean install.  Some of the generated code from this project does not get include in the build.  This causes errors like "cannot find symbol class ASTAxisId"
```

Comment:
To differentiate it from the surrounding text, you can surround mvn clean install with backticks like so:
```
``mvn clean install``
```
This makes it a little clearer that this is the exact command to run.